src/main/java/by/exadel/internship/service/impl/InternshipServiceImpl.java

```diff
@@ -131,6 +132,20 @@ public UserInternshipDTO saveInternship(UserInternshipDTO internshipDTO) {
         return userInternshipDTO;
     }
 
+    public void addUser(UUID userId, UUID internshipId) {
+        userRepository.findById(userId)
+                .orElseThrow(() -> new NotFoundException("User Not Found"));
+
+        internshipRepository.findById(internshipId)
+                .orElseThrow(() -> new NotFoundException("Internship Not Found"));
+
+        int exist = internshipRepository.checkUserExists(userId, internshipId);
```

Comment:
Why do you double check user?

Reply:
1st time we check user 
2nd time we check if our user already exist with our internship table

Reply:
rename method `checkUserExists` to more appropriate name
For example, `checkUserAssign`

Reply:
done